bin/newman.js

```diff
@@ -85,6 +85,42 @@ program
         });
     });
 
+// The `request` command allows you to run single request with the provided options.
+program
+    .command('request <url>')
+    .description('Run single request with provided options')
+    .usage('<url> [options]')
+    .option('-X, --request <method>',
+        'Specify request method')
+    .option('-H, --header <header>',
+        'Specify request header', util.cast.memoize, [])
+    .option('-A, --user-agent <agent>',
+        'Specify request user agent')
+    .option('-d, --data <data>',
```

Comment:
Just a caveat to be aware of while handling this option, if no method is explicitly provided while using this option `-d`, the curl defaults to `POST` method, otherwise the provided method will be used.
```sh
# would resolve in PUT
$ curl -X PUT -d \'{"key":"value"}\' https://google.com 

# would resolve in POST
$ curl -d \'{"key":"value"}\' https://google.com 
```
**Note:** This applies to all the body/data related options.

Reply:
Noted.